config/crd/bases/config.tanzu.vmware.com_clientconfigs.yaml

```diff
@@ -103,8 +103,8 @@ spec:
                           image:
                             description: Image is an OCI compliant image. Which include
                               DNS-compatible registry name, a valid URI path(MAY contain
-                              zero or more ‘/’) and a valid tag. Contains a manifest
-                              file E.g., harbor.my-domain.local/tanzu-cli/plugins-manifest:latest
+                              zero or more ‘/’) and a valid tag. E.g., harbor.my-domain.local/tanzu-cli/plugins-manifest:latest
+                              Contains list of CLIPlugin API resources
```

Comment:
**nitpick**: As above, can this be more precise? (How are the CLIPlugin API resources contained within the OCI image?)